cognee/tests/integration/test_relational_db_migration.py

```
@@ -0,0 +1,138 @@
+            found_edges.add((source_name, target_name))
+            distinct_node_names.update([source_name, target_name])
+
+    elif graph_db_provider == "kuzu":
+        query_str = f"""
+        MATCH (n:Node)-[r:EDGE]->(m:Node)
+        WHERE r.relationship_name = '{relationship_label}'
+        RETURN r, n, m
+        """
+        rows = await graph_engine.query(query_str)
+        for row in rows:
+            r_data = row[0]
```

Ruff (0.8.2)

Local variable `r_data` is assigned to but never used

Remove assignment to unused variable `r_data`

(F841)